src/main/java/com/iexec/common/utils/TestUtils.java

```diff
@@ -0,0 +1,67 @@
+package com.iexec.common.utils;
+
+import java.math.BigInteger;
+
+import com.iexec.common.chain.ChainDeal;
+import com.iexec.common.chain.ChainTask;
+import com.iexec.common.chain.ChainTaskStatus;
+import com.iexec.common.chain.ContributionAuthorization;
+import com.iexec.common.security.Signature;
+
+import org.web3j.utils.Numeric;
+
+public class TestUtils {
+
+    // pool credentials
+    public static final String POOL_ADDRESS = "0xc911f9345717ba7c8ec862ce002af3e058df84e4";
+    public static final String POOL_PRIVATE = "102522099393445372136873302035232363929034445624871422932329330474318520724961";
```

Comment:
Can you put plain hex? (Much commonly used)

Reply:
Sure yes, thanks!